generators/app/templates/src/main/java/package/web/rest/_JaversEntityAuditResource.java

```diff
@@ -0,0 +1,136 @@
+package <%=packageName%>.web.rest;
+
+import <%=packageName%>.domain.EntityAuditEvent;
+import <%=packageName%>.web.rest.util.PaginationUtil;
+import org.javers.core.Javers;
+import org.javers.core.metamodel.object.CdoSnapshot;
+import org.javers.repository.jql.QueryBuilder;
+import <%=packageName%>.security.AuthoritiesConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.security.access.annotation.Secured;
+import com.codahale.metrics.annotation.Timed;
+
+import javax.inject.Inject;
+import java.net.URISyntaxException;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ * REST controller for getting the audit events for entity
+ */
+@RestController
+@RequestMapping("/api")
+public class JaversEntityAuditResource {
+
+    private final Logger log = LoggerFactory.getLogger(JaversEntityAuditResource.class);
+
+    @Inject
+    private Javers javers;
+
+
+    /**
+     * fetches all the audited entity types
+     *
+     * @return
+     */
+    @RequestMapping(value = "/audits/entity/all",
+        method = RequestMethod.GET,
+        produces = MediaType.APPLICATION_JSON_VALUE)
+    @Timed
+    @Secured(AuthoritiesConstants.ADMIN)
+    public List<String> getAuditedEntities() {
+
+      return Arrays.asList(<%- auditedEntities %>);
```

Comment:
Oh sorry seems I forgot it, but I thought of rewriting the file when adding a new entity, will try to add it during the day.